README.md

```diff
@@ -25,9 +25,17 @@ In this challenge use `Test Driven Development` to build a RESTful API using Nod
 Demonstrate your understanding of this week's concepts by answering the following free-form questions. Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager.
 
 1. In Jest, what are the differences between `describe()` and `it()` globals, and what are good uses for them?
+
+-  You can declare/break up tests by calls to it(), nested inside a call to describe().
+
 1. What is the point of `Test Driven Development`? What do you think about this approach?
+
+- Test Driven Development or TDD, relies on repetitve short term development cycles to ensure that as features are added toan application very specific test hwlp improve software reliability. This minimizes risk for bugs that can be tough debug because it is so imbeded in the code. I think this is a great design for simple applications and confidence.
```

Comment:
The bit at the end of this first sentence is a bit hard to read; be sure to proof-read answers if you have time (although this sounds pretty on the mark)!